utils/uhubctl/Makefile

```diff
@@ -0,0 +1,37 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=uhubctl
+PKG_VERSION:=2.2.0
+PKG_RELEASE:=1
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://github.com/mvp/uhubctl/archive/v$(PKG_VERSION)
+PKG_HASH:=e5a722cb41967903bedbab4eea566ab332241a7f05fc7bc9c386b9a5e1762d8b
+PKG_CAT:=zcat
```

Comment:
are you sure that's necessary? It seems unlikely for a plain github archive.

Reply:
This is a codeload archive. So no.

Reply:
I didn't realise it was optional, as I was just following the examples at https://openwrt.org/docs/guide-developer/packages. I'll remove this. 👍

Reply:
Yeah, that example was trying to show off too many things, I've removed it.